src/game/Creature.cpp

```diff
@@ -1831,7 +1834,7 @@ void Creature::SendAIReaction(AiReaction reactionType)
 void Creature::CallAssistance()
 {
     // FIXME: should player pets call for assistance?
-    if (!m_AlreadyCallAssistance && getVictim() && !isCharmed())
+    if (!m_AlreadyCallAssistance && AllowCallAssistance() && getVictim() && !isCharmed())
```

Comment:
Current cmangos nomenclature can use either CanCallAssistance or IsAllowedToCallAssistance so that we stay consistent with getters.